plone/app/content/tests/test_widgets.py

```diff
@@ -459,6 +473,8 @@ def testFileUpload(self):
         chunk.filename = 'test.xml'
         self.request.form['file'] = chunk
         self.request.REQUEST_METHOD = 'POST'
+        # the next calls plone.app.dexterity.factories and does a
+        # transaction.commit. Needs cleanup and FunctionalTesting layer.
```

Comment:
That's where the commit happens: https://github.com/plone/plone.app.dexterity/blob/master/plone/app/dexterity/factories.py#L74